.github/workflows/integration.yml

```diff
@@ -8,7 +8,7 @@ on:
       - release-*
   pull_request:
 env:
-  GO_VERSION: '1.21'
```

Comment:
it was my understanding this would get us the latest version of go of this release branch

Reply:
I noticed that the action has the following set:

```
Run actions/setup-go@v4
  with:
    go-version: 1.21.1
    check-latest: false
    token: ***
    cache: true
  env:
    GO_VERSION: 1.21.1
    NIX_VERSION: 2.15.1
```

The `check-latest: false` especially combined with `go-version` per:

- [Check latest version](https://github.com/actions/setup-go#check-latest-version)
- [Getting go version from the go.mod file](https://github.com/actions/setup-go#getting-go-version-from-the-gomod-file)

I wasn't sure if this would pick the latest patch release rather than the same - **1.21.0**.